### DISCUSSION TITLE
Hiding "Quality monitor" overlay on mouseover

### DISCUSSION THREAD

**Discussion by fragtion** [rustdesk/rustdesk#6380] [Ideas]
I prefer to show the "Quality monitor" most of the time, as it's good to know the latency (Delay), Codec, Bitrate etc.  However, that overlay can be very annoying and obscures/gets in the way of what lies beneath it (at arguably a very important position where Windows shows full-screen window controls).
What if we could simply hide the overlay on hover/mouseover? If we could also move it around somehow that would also be nice :D

**Comment by IVoyt** [comment]
Up. That's very annoying.
I wonder why developers still didn't implement this

**Reply by open-trade** [reply]
I do not think it is hard to implement this with vibe coding. 

Many users has submitted pull requests with vibe coding to this repo.

I wonder why you still didn't implement this.

**Reply by IVoyt** [reply]
> I do not think it is hard to implement this with vibe coding.
> 
> Many users has submitted pull requests with vibe coding to this repo.
> 
> I wonder why you still didn't implement this.

Take it easy, take a pill. What triggered you? Vibe code if you want to and like it, no one stops you from this

**Comment by Loremipsumdolorsitametconsetetur** [comment]
+1
We too would appreciate this being a thing for our custom RustDesk clients.
